modules/caddyhttp/server.go

```diff
@@ -1124,34 +1124,3 @@ const (
 	// For tracking the real client IP (affected by trusted_proxy)
 	ClientIPVarKey string = "client_ip"
 )
-
-var networkTypesHTTP3 = map[string]string{
-	"unixgram": "unixgram",
-	"udp":      "udp",
-	"udp4":     "udp4",
-	"udp6":     "udp6",
-	"tcp":      "udp",
-	"tcp4":     "udp4",
-	"tcp6":     "udp6",
-	"fdgram":   "fdgram",
-}
-
-// RegisterNetworkHTTP3 registers a mapping from non-HTTP/3 network to HTTP/3
-// network. This should be called during init() and will panic if the network
-// type is standard, reserved, or already registered.
-//
-// EXPERIMENTAL: Subject to change.
-func RegisterNetworkHTTP3(originalNetwork, h3Network string) {
```

Comment:
Moving this will be a breaking change to https://sourcegraph.com/github.com/tailscale/caddy-tailscale/-/blob/module.go?L36-39